src/components/learner-credit-management/members-tab/GroupMembersCsvDownloadTableAction.jsx

```diff
@@ -11,6 +11,10 @@ import { useBudgetId, useSubsidyAccessPolicy } from '../data';
 const GroupMembersCsvDownloadTableAction = ({
   tableInstance,
 }) => {
+  let buttonNumRelevantRows = tableInstance.itemCount;
+  if (tableInstance?.selectedFlatRows.length > 0) {
```

Comment:
[nit/suggestion] I think instead of grabbing the `tableInstance.itemCount`  `tableInstance.selectedFlatRows` and checking for the length, we can check for the `tableInstance.state.selectedRowIds`.

it would look something like this:

`const selectedEmails = Object.keys(tableInstance.state.selectedRowIds);`
 `const selectedEmailCount = selectedEmails.length;`